developer/src/kmc-kmn/test/test-compiler.ts

```diff
@@ -1,29 +1,16 @@
 import 'mocha';
-import sinon from 'sinon';
-import chai, { assert } from 'chai';
-import sinonChai from 'sinon-chai';
+import { assert } from 'chai';
 import { KmnCompiler } from '../src/main.js';
 import { dirname } from 'path';
 import { fileURLToPath } from 'url';
 import fs from 'fs';
 import { TestCompilerCallbacks } from '@keymanapp/developer-test-helpers';
 
 const __dirname = dirname(fileURLToPath(import.meta.url)).replace(/\\/g, '/');
-const baselineDir = __dirname + '/../../../../../common/test/keyboards/baseline/';
-chai.use(sinonChai);
+const keyboardsDir = __dirname + '/../../../../../common/test/keyboards/';
```

Comment:
Shouldn't this be:
```suggestion
const keyboardsDir = __dirname + '/../../../../common/test/keyboards/';
```
?

Reply:
This is relative to the build folder not the src folder, so it's one level deeper. Messy, yep.